students/Kate_Koehn/Session06/mailroom_test.py

```diff
@@ -0,0 +1,92 @@
+"""
+tests for mailroom part 4
+"""
+
+
+import pytest
+import mailroom_4 as mr
+
+
+def get_sample_donors():
+    donors = {"Rick Sanchez": [3.00, 1.00],
+              "Liz Lemon": [4000.00, 3000.00, 6000.00],
+              "Andy Dwyer": [10.00],
+              "Brendan Small": [3.00, 10.00],
+              "Coach McGuirk": [2.00, 1.00]}
+    return donors
+
+
+mr.donors = get_sample_donors()
+
+
+def test_adds_new_donor():
+    mr.add_new_donor("Jake Ham")
+
+    assert "Jake Ham" in mr.donors
```

Comment:
I think you have a function that checks if a name is in the db, yes? then good to use that, in case the structure of the db changes.